modules/tip/logic/tipLogic.js

```diff
@@ -206,21 +214,14 @@ const TipLogic = {
   async updateTipsDB(remoteTips) {
     await lock.acquire('TipLogic.updateTipsDB', async () => {
       const localTips = await Tip.findAll({ raw: true });
-      const remoteTipIds = [...new Set(remoteTips.map(tip => tip.id))];
       const localTipIds = [...new Set(localTips.map(tip => tip.id))];
 
-      const newTipsIds = remoteTipIds.filter(id => !localTipIds.includes(id));
-      const oldTipsIds = remoteTipIds.filter(id => localTipIds.includes(id));
+      const newTips = remoteTips.filter(({ id }) => !localTipIds.includes(id));
 
       // Send appropriate notifications for new tips
-      await newTipsIds.asyncMap(id => NotificationLogic.handleNewTip(remoteTips.find(tip => tip.id === id)));
-      await oldTipsIds.asyncMap(id => NotificationLogic.handleOldTip(
-        localTips.find(tip => tip.id === id),
-        remoteTips.find(tip => tip.id === id),
-      ));
-
-      const result = await newTipsIds.asyncMap(async id => {
-        const tip = remoteTips.find(t => t.id === id);
+      await newTips.asyncMap(NotificationLogic.handleNewTip);
```

Comment:
same as with claims, lets handle this in the `TipLogic.insertTips` inserted result